specs/Deployment/DeploymentAPI.md

```diff
@@ -42,12 +42,13 @@ are already present on the system, or that the version present is the minimum ve
 the application. Instead, the Deployment API should be used to validate that the WinAppSDK is at the
 correct version and that all required packages are present.
 
-The Deployment API addresses the following two developer scenarios:
+The Deployment API addresses the following three developer scenarios:
```

Comment:
SUGGESTION: Remove 'three'. Kinda redundant with the actual list, and if the list changes (again) it's one less edit to have to make